docs/source/architecture/poet.rst

```diff
@@ -594,7 +595,7 @@ providing username and ledgername as input parameters to
 ``generateSignUpData()`` and ``unsealSignUpData()``. Then the sign-up tuple 
 (username, ledgername, PPK, PSK, MCID) is sealed to disk, with username and
 ledgername used to generate the filename. Anytime a user authenticates to the
-service the latter can have the enclave unseal and use the sign-up tuple from
+server the latter can have the enclave unseal and use the sign-up tuple from
```

Comment:
I think we should leave it to `service`, a process running on a machine. Not a `server` as in web terminology.

Reply:
I agree.